# The Formation of Planetary Systems

## Angular Momentum

Recall that linear momentum is the combination of mass $m$ with a velocity $v$ (i.e., $p = m\times v = mv$).  This mostly described motion along a straight line.  Let's now apply momentum to circular motion, which requires us to consider how a mass moves on a circle of constant radius $r$.  The simplest form is to define the angular momentum $L$ as:

\begin{align}
L &= m\times v\times r = mvr,
\end{align}

which depends on the mass $m$, velocity $v$ and the radius $r$.  Note that the combination of the mass and radius is also referred to as the mass distribution, or how spread out the mass is.

The angular momentum is a conserved quantity, which means that initial amount has to equal the final amount.  This is why a figure skater spins faster as she pulls in her arms and slower when she kicks out her leg to land.

Let's apply the relationship of angular momentum to Jupiter.  Jupiter is orbiting the Sun and spinning.  Jupiter will have two parts to its angular momentum: orbital and spin. 



### Jupiter's Orbital Angular Momentum
Here we're assuming Jupiter to have a circular orbit, although its real orbit is elliptical.  Recall from [Gravity and Orbits](https://saturnaxis.github.io/IntroAstro/Tutorials/gravity-and-orbits.html#gravity-and-orbits) that we determined Jupiter's mass using one of its moons, Europa.  Therefore, we know Jupiter's mass is $9.54 \times 10^{-4}\ M_\odot$.  We also know that Jupiter's semimajor axis is $5.2\ {\rm AU}$ (see the [NASA Fact Sheet](https://nssdc.gsfc.nasa.gov/planetary/factsheet/jupiterfact.html)).

We can determine Jupiter's orbital speed using the equation for circular velocity, but using our special units of $M_\odot$, AU, and years.  The circular velocity is then

\begin{align}
v_c &= \sqrt{\frac{GM}{a}} = \sqrt{\frac{GM_\odot}{a_{Jup}}} = \sqrt{\frac{4\pi^2}{5.2}}, \\
&= 2.8\ \frac{\rm AU}{\rm yr}.
\end{align}


In [12]:
import numpy as np

G = 4*np.pi**2 #value for G using Msun, yr, and AU as units
a_Jup = 5.204 #semimajor axis of Jupiter in AU
M = 1 # solar mass

v_c = np.sqrt(G*M/a_Jup)
print("The circular velocity of Jupiter is %1.3g AU/yr" % v_c)

The circular velocity of Jupiter is 2.75 AU/yr



**Does the above result make sense?**  The circumference of Jupiter's orbit $C_{Jup}$ is $2\pi\times (5.2\ {\rm AU}) = 32.67\ {\rm AU}$ and Jupiter's orbital period $P_{Jup}$ is $11.86\ {\rm yr}$.  We can simply divide the circumference by the orbital period to determine the orbital velocity $v_c = C_{Jup}/P_{Jup} = 2.76\ {\rm AU/yr}$.  

Now we can determine Jupiter's orbital angular momentum $L_{Jup}^{orb}$ as

\begin{align}
L_{Jup}^{orb} &= \left(m v_{c}r\right)_{Jup} = \left(9.54\times 10^{-4}\ {\rm M_\odot} \right) \left(2.76\ {\rm AU/yr}\right) \left(5.2\ {\rm AU} \right), \\
& = 1.37 \times 10^{-2}\ \frac{M_\odot\ \rm AU^2}{\rm yr}.
\end{align}

In [13]:
import numpy as np

M_Jup = 9.54e-4 #mass of Jupiter in solar masses

L_Jup = M_Jup * v_c * a_Jup
print("The angular momentum of Jupiter is: %1.3g M_sun AU^2/yr." % L_Jup)

The angular momentum of Jupiter is: 0.0137 M_sun AU^2/yr.


This result by itself doesn't give us much context.  Instead of just computing a value, we compare the orbital angular momentum of Earth to Jupiter through a ratio.  We expect Jupiter to have more angular momentum because it's more massive and has a larger orbit.  Therefore we compure the ratio as

\begin{align}
\frac{L_{Jup}^{orb}}{L_{\oplus}^{orb}} &= \frac{\left(m v_{c} r\right)_{Jup}}{\left(mv_{c} r \right)_{\oplus} }, \\
&= \left(\frac{m_{Jup}}{m_\oplus}\right) \left(\frac{v_{c,Jup}}{v_{c,\oplus}}\right)\left(\frac{a_{Jup}}{a_\oplus}\right).
\end{align}

From the [NASA Fact Sheet](https://nssdc.gsfc.nasa.gov/planetary/factsheet/jupiterfact.html), we know that Jupiter is approximately $317\ m_\oplus$, or $317\times$ more massive than the Earth.  Earth's semimajor axis is $1\ {\rm AU}$, and circular velocity is $2\pi\ {\rm AU/yr}$.  By substitution, we find

\begin{align}
\frac{L_{Jup}^{orb}}{L_{\oplus}^{orb}} &= \left(\frac{317\ m_\oplus}{m_\oplus}\right) \left(\frac{2.75\ {\rm AU/yr}}{2\pi\ {\rm AU/yr}}\right) \left(\frac{5.2\ {\rm AU}}{1\ {\rm AU}}\right)
= 723. 
\end{align}

This means that Jupiter has $723\times$ more orbital angular momentum than the Earth.

```{note}
Objects on larger orbits typically have more angular momentum than those on smaller orbits.  If you move an object inward it has to orbit faster to conserve angular momentum.  When this is a gas cloud, it leads to the cloud spinning up.
```

In [16]:
mass_ratio = 317 #number of Earth masses in Jupiter
v_ratio = v_c/(2*np.pi) #orbital velocity ration between Earth & Jupiter (circular orbits)

L_ratio = mass_ratio*v_ratio*a_Jup
print("Jupiter has %3.3g times more angular momentum than Earth." % L_ratio)

Jupiter has 723 times more angular momentum than Earth.



### Jupiter's Spin Angular Momentum
The spin angular momentum is a little more complicated because we need to include the individual angular momentum from every tiny mass element.  This combines to form the mass distribution.  For Jupiter, we can assume that the planet is roughly spherical to simplify the problem.  For this tutorial, we'll mostly skip to the final form where interested students can see an example from [OpenStax](https://openstax.org/books/college-physics-2e/pages/10-5-angular-momentum-and-its-conservation) for more details.

For a uniform sphere the spin angular momentum, we have

\begin{align}
L_{Jup}^{spin} = \frac{2}{5} m_{Jup}R_{Jup}^2 \omega_{Jup},
\end{align}

where the radius of Jupiter is ${\sim}11R_\oplus$ and $1R_\oplus = 4.263 \times 10^{-5}\ {\rm AU}$.  The term $\omega_{Jup}$ refers to its rotational speed $(=2\pi/P_{rot})$, which we can calculate using its rotational period $P_{rot}$ that is ${\sim}10\ {\rm hr} = 1.25 \times 10^{-3}\ {\rm yr}$.  Then the above equation becomes

\begin{align}
L_{Jup}^{spin} &= \frac{2}{5}\left(9.54\times 10^{-4}\ M_\odot \right) \left(4.69\times 10^{-4}\ {\rm AU}\right)^2 \left(\frac{2\pi}{1.25 \times 10^{-3}\ {\rm yr}} \right), \\
&= 4.20 \times 10^{-7}\ \frac{M_\odot\ \rm AU^2}{\rm yr}.
\end{align}

We can see that Jupiter's spin angular momentum is a lot smaller than its orbital angular momentum.  Both expressions for angular momentum depend on the distance, but to a different degree.  The orbital angular momentum depends only linearly on $r$, where the spin angular momentum depends on $r^2$.  Coupled with our previous finding that smaller $r$ leads to lower orbital angular momentum, this is even more dramatic for the spin angular momentum.

Again let's compare the spin angular momentum between Jupiter and the Earth by using a ratio.  We can construct the ratio as

\begin{align}
\frac{L_{Jup}^{spin}}{L_\oplus^{spin}} &= \frac{2}{5} m_{Jup}R_{Jup}^2 \left(\frac{2\pi}{P_{Jup}^{rot}} \right) \times \frac{5}{2} \frac{1}{m_{\oplus}R_{\oplus}^2} \left(\frac{P^{rot}_\oplus}{2\pi} \right), \\
&= \left(\frac{m_{Jup}}{m_\oplus}\right)\left(\frac{R_{Jup}}{R_\oplus}\right)^2 \left(\frac{P_\oplus^{rot}}{P_{Jup}^{rot}} \right), \\
& = \left(\frac{317\ m_\oplus}{m_\oplus}\right) \left(\frac{11\ R_\oplus}{R_\oplus}\right)^2 \left(\frac{24\ {\rm hr}}{11\ {\rm hr}}\right), \\
&= 8.37 \times 10^4.
\end{align}

Jupiter has ${\sim}84,000\times$ more spin angular momentum than the Earth.  We expect this because Jupiter is more massive, bigger, and spins faster.


In [30]:
import numpy as np
R_E = 4.263e-5 #Radius of Earth in AU
R_Jup = 11*R_E #Radius of Jupiter in AU
PJ_rot = 11/24/365.25 #Jupiter's rotation in years
PE_rot = 1./365.25 #Earth's rotation period in years

L_Jup_spin = 0.4*M_Jup*R_Jup**2*(2*np.pi/PJ_rot)
print("The spin angular momentum of Juptier is: %1.3g Msun AU^2/yr" % L_Jup_spin)

L_spin_ratio = mass_ratio*(R_Jup/R_E)**2*(PE_rot/PJ_rot)
print("Jupiter has approximately %1.3g times more spin angular momentum than Earth." % L_spin_ratio)

The spin angular momentum of Juptier is: 4.2e-07 Msun AU^2/yr
Jupiter has approximately 8.37e+04 times more spin angular momentum than Earth.
